website/www/site/content/en/blog/beam-2.20.0.md

```diff
@@ -43,7 +41,6 @@ Python SDK: . (#10223).
 * [BEAM-8841](https://issues.apache.org/jira/browse/BEAM-8841) Added ability to write to BigQuery via Avro file loads
 * [BEAM-9228](https://issues.apache.org/jira/browse/BEAM-9228) Direct runner for FnApi supports further parallelism
 * [BEAM-8550](https://issues.apache.org/jira/browse/BEAM-8550) Support for @RequiresTimeSortedInput in Flink and Spark
-* [BEAM-6857](https://issues.apache.org/jira/browse/BEAM-6857) Added support for dynamic timers
```

Comment:
Why was this removed?

Reply:
@robertwb: Oh, thanks for pointing out. It was updated recently. I have synced this PR to current repository. 🙂